x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/DatafeedJobsIT.java

```diff
@@ -106,15 +108,15 @@ public void testDatafeedTimingStats_DatafeedRecreated() throws Exception {
         Instant now = Instant.now();
         indexDocs(logger, "data", numDocs, now.minus(Duration.ofDays(14)).toEpochMilli(), now.toEpochMilli());
 
-        Job.Builder job = createScheduledJob("lookback-job");
+        Job.Builder job = createScheduledJob("lookback-job-datafeed-recreated");
 
-        String datafeedId = "lookback-datafeed";
+        String datafeedId = "lookback-datafeed-datafeed-recreated";
         DatafeedConfig datafeedConfig = createDatafeed(datafeedId, job.getId(), Arrays.asList("data"));
 
         registerJob(job);
         putJob(job);
 
-        for (int i = 0; i < 2; ++i) {
+        CheckedRunnable<Exception> process = () -> {
```

Comment:
nit: `process` could be a more descriptive name -> `openAndRunJob`?

Reply:
Done.